frontend/src/graphql/mutations.js

```diff
@@ -144,6 +144,7 @@ export const createContributionMessage = gql`
 export const login = gql`
   mutation($email: String!, $password: String!, $publisherId: Int) {
     login(email: $email, password: $password, publisherId: $publisherId) {
+      gradidoID
       email
```

Comment:
do we need the email as a result of this mutation anymore, when we get the gradidoID now?

Reply:
Next step. Remove the email from the graphql User